sirepo/template/raydata.py

```diff
@@ -48,7 +49,8 @@ def _filename_and_image(path):
 
     def _paths():
         d = _dir_for_scan_uuid(_parse_scan_uuid(data))
-        for f in pkio.sorted_glob(d.join('*.png'), key='mtime'):
+
+        for f in glob(str(d.join('/**/*.png')), recursive = True):
```

Comment:
nit: no spaces around = in  `recursive = True`

Reply:
eb41875ad13fdf2a9ff2f5e261bcf5432c048ace